e2e_tests/data/coco/to/output.json

```diff
@@ -85,13 +85,13 @@
   ],
   "categories": [
     {
-      "id": 3961009249,
-      "name": "test_poly",
+      "id": 348813479,
+      "name": "test_bb",
       "supercategory": "root"
     },
     {
-      "id": 348813479,
-      "name": "test_bb",
+      "id": 3961009249,
+      "name": "test_poly",
       "supercategory": "root"
```

Comment:
This was causing another E2E test failure where conversions to COCO [now output consistent, sorted annotation categories](https://github.com/v7labs/darwin-py/pull/788/commits/6b1560acb4bc68eedc043ec7fc2db979785fbcd4). Previously, they could be generated in different orders from run to run